diff --git a/app/build.gradle b/app/build.gradle
index 6f70e94..2612cc6 100644
--- a/app/build.gradle
+++ b/app/build.gradle
@@ -11,8 +11,8 @@ android {
         applicationId 'dev.besan.browserbrake'
         minSdk 29
         targetSdk 36
-        versionCode 14
-        versionName '0.5.0-alpha4'
+        versionCode 15
+        versionName '0.5.0-alpha5'
     }
 
     def testKey = rootProject.file('browser-brake-test.keystore')
